modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,67 +12,66 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization (including the Federated ones).
-It is the recommended initial solution pending future corporate investments in enterprise specific OAuth 2.0 offerings.
+The Mule OAuth 2.0 Provider is an alternative from MuleSoft for the OAuth 2.0 provider, and can be used in any API Platform organization, including the Federated ones. The Mule OAuth 2.0 provider is the first-step solution for your OAuth 2.0 offerings.
```

Comment:
The change changes the meaning. "OAuth 2.0 provider alternative" is no the same as "an alternative from MuleSoft for the OAuth 2.0 provider". In particular there is no canonical OAuth 2.0 provider as the *the* seems to imply.

Reply:
Changed a bit. Kim please verify.